nebula/ui/components/forms/VPNComboBox.qml

```diff
@@ -57,6 +57,7 @@ ComboBox {
 
     delegate: ItemDelegate {
         id: comboDelegate
+        objectName: "category-Dropdown-Option-" + index
```

Comment:
This cannot be called category because it's a shared component. Maybe it should use the root obj name?